yarn.lock

```diff
@@ -16,14 +16,12 @@
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```

Comment:
I have no idea why Yarn dropped these out. My v1.3 might've just been too old? I'll re-do after yarn upgrade.

Reply:
I'm least concerned about yarn.lock. It'll get rebuilt soon enough.

Reply:
Fair enough! :-)